### PR TITLE
Async calls to geoserver

### PR DESCRIPTION
**Pull request by jkburges**
Previously, a deprecation warning was happening due to the synchronous calls.

### CODE REVIEW

**Comment by jkburges** [comment]
@lbesnard let me know when you want to smoke test this change.

File: grails-app/views/finder/_auv_functions_js.gsp [hunk 1]
```diff
@@ -311,13 +309,13 @@
                 + "&version=1.0.0&maxfeatures=1&sortby=" + variable + "+a";
 
 
-        var xmlDoc = getXML(url);
+        getXML(url, function(doc) {
+            var xmlDoc = doc;
```
Comment (dnahodil) [inline grails-app/views/finder/_auv_functions_js.gsp]: Does this function do anything? I would think that the `xmlDoc` variable is only in scope within this function, meaning that the function doesn't really do anything.

Comment (jkburges) [inline grails-app/views/finder/_auv_functions_js.gsp]: Ah yes, I meant to write a comment about this, as I saw this too.  In short, "probably not" - but I didn't want to go changing too much.  There are some interesting scoping issues going on in this javascript.

Comment (dnahodil) [inline grails-app/views/finder/_auv_functions_js.gsp]: Sure, no problems.

**Pull request merged**